Vonage/client-sdk-video-macos
# VIDCS-2118: Add App store privacy manifest to Mac SDK 2.28.0

#### What is this PR doing?
Adds tflite and privacy files in a Resources folder like 
<img width="345" alt="image" src="https://github.com/Vonage/client-sdk-video-macos/assets/4369083/ed306a35-c7af-4330-ac10-38cffe48c82e">


#### How should this be manually tested?
Make sure blur transfomer works (it should not have worked before)

For testing Privacy reports follow the steps at https://github.com/opentok/native-sdk-src/pull/4542#issue-2223861844


#### What are the relevant tickets?

[VIDCS-2118](https://jira.vonage.com/browse/VIDCS-2118)
